modules/light-clients/08-wasm/types/proposal_handle.go

```diff
@@ -28,7 +28,7 @@ func (cs ClientState) CheckSubstituteAndUpdateState(
 
 	_, ok := substituteClient.(*ClientState)
 	if !ok {
-		return errorsmod.Wrapf(
```

Comment:
we could remove the `fmt.Sprintf()` below too :D 

And then actually use `Wrapf()` here correctly!

Reply:
glorious

Reply:
love the commit msg 😆